src/main/java/net/earthcomputer/clientcommands/features/PlayerRandCracker.java

```diff
@@ -337,6 +357,39 @@ public static long getSeed(Random rand) {
         }
     }
 
+    public enum ThrowItemsState {
+        NOT_ENOUGH_ITEMS(false, new TranslatableText("playerManip.notEnoughItems")),
+        NOT_POSSIBLE(false, new TranslatableText("playerManip.throwError")),
+        UNKNOWN_SEED(false, new TranslatableText("commands.cenchant.uncracked")),
+        SUCCESS(true),
+        ;
+
+        private final Boolean success;
+        private TranslatableText message;
+
+        ThrowItemsState(Boolean success) {
+            this(success, null);
+        }
+
+        ThrowItemsState(Boolean success, TranslatableText message) {
+            this.success = success;
+            this.message = message;
+        }
+
+        public TranslatableText getMessage() {
+            return this.message;
+        }
+
+        public ThrowItemsState setMessageArgs(Object... args) {
```

Comment:
Enums shouldn't be mutable. If you need to store any state, use a non-enum class. Sorry, I did not realize you wanted to store state.
```java
public static class ThrowItemsResult {
  private final Type type;
  private final TranslatableText message;

  public ThrowItemsResult(Type type, Object... args) {
    this.type = type;
    this.message = new TranslatableText(type.getTranslationKey(), args);
  }

  public Type getType() {
    return type;
  }

  public TranslatableText getMessage() {
    return message;
  }

  public enum Type {
    NOT_ENOUGH_ITEMS(false, "playerManip.notEnoughItems"),
    NOT_POSSIBLE(false, "playerManip.throwError"),
    UNKNOWN_SEED(false, "commands.cenchant.uncracked"),
    SUCCESS(true, null),
    ;

    private final boolean success;
    private final String translationKey;

    Type(boolean success, String translationKey) {
      this.success = success;
      this.translationKey = translationKey;
    }

    public boolean isSuccess() {
      return success;
    }

    public String getTranslationKey() {
      return translationKey;
    }
  }
}
```